hack/jenkins/linux_integration_tests_kvm.sh

```diff
@@ -25,23 +25,25 @@
 
 set -e
 
+OS_ARCH="linux-amd64"
+
 # Copy only the files we need to this workspace
-mkdir -p out/
-gsutil cp gs://minikube-builds/${MINIKUBE_LOCATION}/minikube-linux-amd64 out/
-gsutil cp gs://minikube-builds/${MINIKUBE_LOCATION}/e2e-linux-amd64 out/
-gsutil cp -r gs://minikube-builds/${MINIKUBE_LOCATION}/testdata .
+mkdir -p out/ testdata/
+gsutil cp gs://minikube-builds/${MINIKUBE_LOCATION}/minikube-${OS_ARCH} out/
```

Comment:
Hmm, would you mind moving this up to a function inside a new "common.sh" file that we source in each test? Then OS_ARCH could be a parameter.